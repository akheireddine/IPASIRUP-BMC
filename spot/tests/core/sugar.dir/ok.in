X[4]a
X[3!]a
G[2:4]a
G[4:2]a
F[2:4]a
F[4:2]a
F[2:$]a
F[2..]a
F[2:$!]a
F[2:3!]a
X [4]a | b
G [2:4] a | b
G [4:2] a | b
G [2:] a | b
G [2..] a | b
G [2..!] a | b
G [2..3!] a | b
F [2:4] a |  b
F [4:2]a | F[2:2]b
F[]a|G[]b|X[]c
{a ##0 b ##1 c ##2 d}|->e
{a* ##0 b* ##1 c* ##2 d*}|->e
{(##2 a)[*] ##1 b}|->e
{a ##[0:3] b}|->e
{a* ##[0:3] b}|->e
{a ##[0:3] b*}|->e
{a* ##[0:3] b*}|->e
{a ##[1..] b}|->e
{a ##[:] b}|->e
{a ##[:1] b}|->e
{##[..3] b}|->e
{##[0..2] b*}|->e
{##[1..2] b*}|->e
{a ##[+] b}|->e
{##[*] b}|->e
{(!a)[*];a;b}!
{(!a)[+];a;b}!
{(!a)[*3];a;b}!
{((!a)[*];a)[*2..3];b}!
