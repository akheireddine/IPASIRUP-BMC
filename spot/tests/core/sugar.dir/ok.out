XXXXa
X[!]X[!]X[!]a
XX(a & X(a & Xa))
XX(a & X(a & Xa))
XX(a | X(a | Xa))
XX(a | X(a | Xa))
XXFa
XXFa
X[!]X[!]Fa
X[!]X[!](a | X[!]a)
b | XXXXa
b | XX(a & X(a & Xa))
b | XX(a & X(a & Xa))
b | XXGa
b | XXGa
b | X[!]X[!]Ga
b | X[!]X[!](a & X[!]a)
b | XX(a | X(a | Xa))
XX(a | X(a | Xa)) | XXb
FGa | Gb | XGc
{{a && b};c;1;d}[]-> e
{{a[*]:b[*]};c[*];1;d[*]}[]-> e
{{[*2];a}[*];b}[]-> e
{a:{[*0..3];b}}[]-> e
{{a[*];[*0..3]}:b}[]-> e
{a:{[*0..3];b[*]}}[]-> e
{{a[*]:b[*]} | {a[*];[*0..2];b[*]}}[]-> e
{a;[*];b}[]-> e
{a:{[*];b}}[]-> e
{a:{[*0..1];b}}[]-> e
{[*0..3];b}[]-> e
{[*0..2];b[*]}[]-> e
{[*1..2];b[*]}[]-> e
{a;[*];b}[]-> e
{[*];b}[]-> e
{a[->];b}!
{{!a}[+];a;b}!
{{!a}[*3];a;b}!
{a[->2..3];b}!
